### PR TITLE
(maint) Increase dependency versions

### PR DESCRIPTION
**Pull request by jarretlavallee**
Prior to this PR, there were several major versions of dependency
modules that were released, but not compatible with this module. This
commit increases the maximum version of the dependencies for this
module.

This does not increase puppet-telegraf due to incompatibilities in 4.0.0. There is a commit that will fix this in the next release.

### CODE REVIEW

**Comment by jarretlavallee** [comment]
This is blocked by the litmus tests failing due to the removal of EL 5.

**Comment by MartyEwings** [comment]
Once we merge https://github.com/puppetlabs/puppet_metrics_dashboard/pull/147 this should allow the tests to run

**Review by MartyEwings — APPROVED** [review]
Looks good to me

**Pull request merged**
